Add information about how to add new languages to monaco (#24906)
microsoft/PowerToys: doc/devdocs/common/readme.md

# Common

> The [common](/src/common) folder contains projects that contain code, that are used in multiple projects.

Reviewer: ```suggestion
The [common](/src/common) folder contains projects with code, that is used in multiple projects.
```
